### PR TITLE
Remove references to php -f for occ command

### PR DESCRIPTION
**Pull request by settermjd**
This PR removes all relevant (lingering) references to calling occ using `php -f`, which were pointed out #1989 by @xelan.

### CODE REVIEW

File: modules/admin_manual/pages/release_notes.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ For code cleanup reasons, the execution of background jobs (e.g., for public lin
 
 * If you're using https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html#cron[System cron] to trigger background job execution, there is a new `occ` command (`occ system:cron`) which executes the background jobs. 
   To make use of it, you have to change the entry in `crontab`. 
-  Instead of executing `cron.php` (e.g., `/usr/bin/php -f /path/to/your/owncloud/cron.php`), cron should use `occ system:cron` (e.g., `/usr/bin/php -f /path/to/your/owncloud/occ system:cron`). 
+  Instead of executing `cron.php` (e.g., `/usr/bin/php -f /path/to/your/owncloud/cron.php`), cron should use `occ system:cron` (e.g., `{occ-command-example-prefix} system:cron`).
```
Comment (mmattel) [inline modules/admin_manual/pages/release_notes.adoc]: Shouldn´t the {occ...} also be set in the example?

**Comment by mmattel** [comment]
You may want to check the crontab entry in #1234 to switch to {occ...}

File: modules/admin_manual/pages/release_notes.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/release_notes.adoc]: Sory, but I don't follow what you're asking.

**Comment by settermjd** [comment]
> You may want to check the crontab entry in #1234 to switch to {occ...}

I've updated the example in system_command.adoc on line 13 to use the site-wide variable. Thanks for pointing out that earlier oversight.

File: modules/admin_manual/pages/release_notes.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/release_notes.adoc]: Sorry, you are right. Have overseen that cron.php was not called by occ... 🤦‍♂

**Comment by voroyam** [comment]
This PR also removes the full path to php and occ. 

Either enter the full path or remove the note below the example code block that says "We added the full path on purpose".

I am for entering the full path, that way we can do less things wrong.

I am also for people reading the complete document and then making a change. But with the variables, as I mentioned multiple times, it's very hard to read.

@settermjd

**Pull request merged**